src/lib/TopLevel.hs

```diff
@@ -185,7 +149,6 @@ evalUModule :: TopEnv -> UModule -> TopPassM TopEnv
 evalUModule env untyped = do
   -- TODO: it's handy to log the env, but we need to filter out just the
   --       relevant part (transitive closure of free vars)
```

Comment:
I think that this comment applied to the line you've removed. Maybe remove it too?